src/library/Uno.Material/Styles/Controls/NavigationView.xaml

```diff
@@ -435,16 +435,18 @@
 											<RowDefinition Height="*" />
 										</Grid.RowDefinitions>
 
-										<ContentControl x:Name="HeaderContent"
-														MinHeight="{StaticResource PaneToggleButtonHeight}"
-														IsTabStop="False"
-														Margin="4,4,0,0"
-														Content="{TemplateBinding Header}"
-														ContentTemplate="{TemplateBinding HeaderTemplate}"
-														VerticalContentAlignment="Stretch"
-														HorizontalContentAlignment="Stretch"
-														Style="{StaticResource NavigationViewTitleHeaderContentControlTextStyle}" />
-
+										<Grid Margin="8,0,0,0">
```

Comment:
im curious, what does adding a grid fix? We couldnt fix the cutoff without adding a grid?

Reply:
It's the margin that counts. The content presenter seems to have it's margin set by code, but it's not enough to not get overlapped by the toggle button.